web/themes/joinup/joinup_theme.theme

```diff
@@ -992,3 +1018,18 @@ function joinup_theme_preprocess_table__file_formatter_table(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add theme suggestions directly to the comment body field.
+ */
+function joinup_theme_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (strpos($form['#id'], 'views-exposed-form-search-page-') !== FALSE) {
+    $form['#info']['filter-search_api_fulltext']['label'] = '<span class="search-bar__icon icon icon--search"></span>';
```

Comment:
[System info](https://gist.github.com/anonymous/7d9aa148483b81d37d89c405adbd62da) and chrome i Version 59.0.3071.115 (Official Build) (64-bit), although [FF 54](https://www.mozilla.org/en-US/firefox/54.0/releasenotes/) has the same issue. I'm not logged in.